modules/nf-scil/image/resample/main.nf

```diff
@@ -0,0 +1,60 @@
+process IMAGE_RESAMPLE {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(image)
+
+    output:
+    tuple val(meta), path("*_resampled.nii.gz"), emit: image
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def ref = task.ext.ref ? "--ref" + task.ext.ref : ""
+    def voxel_size = task.ext.voxel_size ? "--voxel_size " + task.ext.voxel_size : ""
+    def volume_size = task.ext.volume_size ? "--volume_size " + task.ext.volume_size : ""
+    def iso_min = task.ext.iso_min ? "--iso_min" + task.ext.iso_min : ""
```

Comment:
iso_min is a boolean flag, not an argument with value. Remove the `+ task.ext.iso_min`